lia-core/build.gradle

```diff
@@ -47,6 +49,11 @@ def getGitHash = { ->
     return stdout.toString().trim()
 }
 
+def getPublishVersionSuffix = { ->
+    return isDebuggable ? "+{$getGitHash()}" : ""
```

Comment:
Shouldn't we prefer to `variant.buildType.isDebuggable()` or `variant.name.contains('snapshot')` instead of using a global variable? We could pass variant as an argument to this method.